### DISCUSSION TITLE
Seeding Values in CRC

### DISCUSSION THREAD

**Discussion by Spice-Weasel** [hex-in/libscrc#20] [Q&A]
Hi there,
I'm trying to use a seed value in calculation for the different algorithms, I can't see any methods that will let me set this. Is there a way around this?
Apologies if this is a trivial question, I'm relatively new to CRC generally.
Thanks

**Comment by hex-in** [comment]
See README.md

Ex:
libscrc.hacker16( b'123456789', poly=0x8005, init=0xFFFF, xorout=0x0000, refin=True, refout=True)
